test/test_main.cpp

```diff
@@ -70,17 +69,32 @@ int main(int argc, char* argv[])
 
     session.config().stream() << "Random Seed: " << session.configData().rngSeed << std::endl;
 
+    if (!qengine && !qfusion && !qunit && !qunit_qfusion) {
+        qfusion = true;
+        qunit = true;
+        qengine = true;
```

Comment:
Qunit_qfusion?

Reply:
We need an option for enabling QUnit just with an intermediate QFusion layer. Enabling them together doesn't let you isolate that layer.

Reply:
Might be worth a comment.

Reply:
Enabling them individually, together, will enable all QUnit tests, all QFusion tests, and QUnit -> QFusion in the intersection. We want to be able to selectively enable just QUnit -> QFusion.